src/Utils/FileSystem.php

```diff
@@ -131,10 +131,11 @@ public static function read(string $file): string
 
 	/**
 	 * Writes a string to a file.
+	 * @param int|NULL $mode
```

Comment:
There should be two spaces after `@param` to align with `@return` and `@throws`.